features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | aanduiding naamgebruik (61.10)       | E           |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 999994037 |
+      | geslachtsaanduiding (04.10) | V         |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                                 | waarde   |
+      | voornamen (02.10)                    | Carolina |
+      | adellijke titel of predicaat (02.20) | BS       |
+      | voorvoegsel (02.30)                  | van der  |
+      | geslachtsnaam (02.40)                | Horn     |
+
+  Rule: wanneer gevraagd wordt om voorletters, wordt voornamen geleverd
+
+    @gba
+    Scenario: met fields is gevraagd om voorletters
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999990160                       |
+      | fields              | naam.voorletters                |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde      |
+      | voornamen | Hendrik Jan |
+
+    @gba
+    Scenario: met fields is gevraagd om de voorletters van de partner
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999990160                       |
+      | fields              | partners.naam.voorletters       |
+      Dan heeft de response een persoon met de volgende 'partner' gegevens
+      | naam           | waarde   |
+      | naam.voornamen | Carolina |
+
+  Rule: wanneer gevraagd wordt om volledigeNaam, wordt voornamen, adellijkeTitelPredicaat, voorvoegsel en geslachtsnaam geleverd
+    
+    @gba
+    Scenario: met fields is gevraagd om volledigeNaam
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999990160                       |
+      | fields              | naam.volledigeNaam              |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                                 | waarde      |
+      | voornamen                            | Hendrik Jan |
+      | adellijkeTitelPredicaat.code         | R           |
+      | adellijkeTitelPredicaat.omschrijving | ridder      |
+      | voorvoegsel                          | de          |
+      | geslachtsnaam                        | Brink       |
+
+  Rule: wanneer gevraagd wordt om aanhef, aanschrijfwijze of gebruikInLopendeTekst, dan worden geslacht, naam en naam van de partner geleverd
+    - de partner worden adellijkeTitelPredicaat, voorvoegsel en geslachtsnaam geleverd
```

Comment:
Deze zin corrigeren. Is een beetje vreemd zoals hij hier staat.

Reply:
inderdaad. "van" aan het begin van de zin toegevoegd